xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -812,6 +823,7 @@ struct MemoEditorDetailModel: ModelProtocol {
             var model = state
             model.backlinks = backlinks
             return Update(state: model)
+                .animation(.easeOutCubic())
```

Comment:
Some transitions became more noticeable with the themed editor/viewer, hence the added `.animation()`s